docs/CONFIGID.md

```diff
@@ -0,0 +1,25 @@
+# mLRS Documentation: Configuration ID #
+
+([back to main page](../README.md))
+
+mLRS allows for 10 different configurations to be stored on the Tx module.  This is useful for when you want to use a single Tx module with multiple receivers and want to use different settings for each receiver.  For example, you may prefer to use 50 Hz mode on your copter but prefer to use 19 Hz mode on your rover. The active configuration ID is set based on the receiver number that is assigned on the MODEL SETUP page within the 'External RF' page/section of EdgeTx/OpenTx. This option is only available when using CRSF or mBridge protocols, SBUS will only use configuration ID 0.
+
+Notes:
+- The selected value for the Tx Ch Source parameter will applied to all configuration IDs.
```

Comment:
just nitpicking
- maybe one should give the values, like "... selected value ("crsf", "in", "mbridge") for ...".
- shouldn't it be "will be applied"

Reply:
Updated to address both items.